src/app/page.tsx

```
@@ -164,38 +186,36 @@ export default function Page() {
-    [claimOrderPayment]
-  );
+      setUserData(undefined);
+    } catch (error: any) {
+      setOpenAlert(true);
+      setAlertText(error.message);
+    }
+  }, [claimOrderPayment]);
 
   useEffect(() => {
-    events && events.length && userData && processPayment(events[0], userData);
+    events && events.length && userData && processPayment();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)